apps/login/src/app/api/idp/start/route.ts

```diff
@@ -14,7 +16,9 @@ export async function POST(request: NextRequest) {
       },
     })
       .then((resp) => {
-        return NextResponse.json(resp);
+        return NextResponse.json(
+          toJson(StartIdentityProviderIntentResponseSchema, resp),
+        );
```

Comment:
**Important:** we must use `toJson/2` when returning a protobuf message as a JSON response since BigInt and other types are not supported by `fetch`